examples/js/web-worker/service-script.js

```diff
@@ -0,0 +1,30 @@
+// const Bugsnag =  require('/node_modules/@bugsnag/js/browser/notifier.js')
+// Bugsnag.start('ed0820ec954fcc35c3bc4f18fd36b206')
+
+const handledErrorBtn = document.querySelector('#handledErrorServWrkr')
+const unhandledErrorBtn = document.querySelector('#unhandledErrorServWrkr')
+
+const registerWorker = () => {
+    if (!('serviceWorker' in navigator)) {
+        console.log('Service Worker not supported')
+        return
+    } else {
+        navigator.serviceWorker.register('/service-worker.js')
```

Comment:
This should solve the service worker registration issue

```suggestion
        navigator.serviceWorker.register('/service-worker.js', { type: 'module' })
```